app/controllers/teachers_controller.rb

```diff
@@ -18,23 +18,27 @@ def new
   def edit; end
 
   def create
-    @new_user = User.new(user_params.merge(password: new_user_temp_password))
-    # TODO: Add support to create students user
-
-    if @new_user.save
-      redirect_to teacher_path(@user),
-                  notice: "#{@new_user.role.capitalize} was created with password #{new_user_temp_password}."
-    else
-      render :new, status: :unprocessable_entity
+    ActiveRecord::Base.transaction do
+      @new_user = User.create(user_params.merge(password: new_user_temp_password))
+      if @new_user.student?
```

Comment:
why would we create a student user in a teachers controller?

Reply:
Make sense. Fixed!